new-frontiers-14/frontier-station-14
# Caduceus atmos update and cloning removal.

## About the PR



This PR brings the Cad's atmos system into compliance by removing the air tanks and introducing a nitrogen and oxygen system with mixer.

It additionally removes the cloning machinery as per Frontier's efforts to scrub mapped cloning systems.

## Why / Balance


N2/O2 compliance.
Shift-start cloning machines are to be removed from Frontier facilities and ships. 

## How to test


Purchase the ship. Ensure cloning facilities are not hiding somewhere. Check atmos. Fans/etc swapped out in another PR.

## Media


![ss+(2024-10-12+at+10 36 42)](https://github.com/user-attachments/assets/3f271ffa-d561-4bbf-9b2e-89bf8f9d093f)



## Requirements



## Breaking changes


**Changelog**


🆑 
- tweak: Updated CAD atmos for n2/o2 compliance.
- remove: Removed cloning from CAD for Frontier compliance.
